charts/tidb-operator/values.yaml

```diff
@@ -12,7 +12,7 @@ rbac:
 timezone: UTC
 
 # operatorImage is TiDB Operator image
-operatorImage: pingcap/tidb-operator:v1.1.0-beta.1
+operatorImage: pingcap/tidb-operator:v1.1.0-beta.2
 imagePullPolicy: IfNotPresent
 
 # tidbBackupManagerImage is tidb backup manager image
```

Comment:
pingcap/tidb-backup-manager should be uncomment and  `pingcap/tidb-backup-manager:v1.1.0-beta.2`?

Reply:
Agree, as from tidb-operator version 1.1, we should keep the backup-related image versions consistent with the tidb-operator version

Reply:
updated, PTAL